public/app/plugins/datasource/tempo/datasource.ts

```diff
@@ -53,8 +54,8 @@ export interface TempoQuery extends DataQuery {
   linkedQuery?: LokiQuery;
   search: string;
   queryType: TempoQueryType;
-  serviceName?: string;
-  spanName?: string;
+  serviceName?: string | SelectableValue;
```

Comment:
I don't think the query should be able to be 'SelectableValue' as the code that uses serviceName/spanName expects them to be strings when running queries.